Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
I just pushed two commits.

One adds Toolchain.sysroot and Toolchain.platformPath on non-OSX platforms where they get a value of nil.

The second commit removes the #if around the use of Toolchain.sysroot in Sources/Build/describe()